csuntechlab/iSTART
# STR-200 -- Implement Illicit Drug Slides

# Description
'Write a meaningful description of your task'
  
# Testing
1. Visit the Illicit Drugs Module.
2. Go through all the slides and ensure there are no...
> misplaced slides
> incorrect spelling
> matches the powerpoint slide
3. Check if all images work properly.
